### ISSUE TITLE
Issuse with step 7 of SKSE64

### ISSUE THREAD

**Issue by Xexman** [spooknik/SkyrimSE-Linux-Modding#1]
On step 7 I pastaed xterm && echo %command% >/dev/null into luach options. I then closed and ran skyrim se and nothing. No terminal no nothing.

**Comment by spooknik** [comment]
You most likely don't have xterm installed then.

What distro and desktop (gnome, KDE, xfce, etc) are you using? You can also replace `xterm` with `gnome-terminal` or whatever program name of your terminal application might be.

**Comment by Xexman** [comment]
Manjaro on Xfce desktop

**Comment by spooknik** [comment]
Okay, so you need to paste this into launch options:

`xfce4-terminal && echo %command% >/dev/null `

**Comment by Xexman** [comment]
Cool that fixxed that issue, now i have  ERROR: ld.so: object '/home/manjaro3/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
pid 6521 != 6518, skipping destruction (fork without exec?)
ERROR: ld.so: object '/home/manjaro3/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.

**Comment by spooknik** [comment]
As I've written in the guide, this error can be safely ignored. You should be able to type whatever commands you want into the terminal.

**Comment by Xexman** [comment]
Sorry i missed that now  ~/".steam/root/compatibilitytools.d/skyrim-proton/proton" run skse64_loader.exe
bash: /home/manjaro3/.steam/root/compatibilitytools.d/skyrim-proton/proton: No such file or directory
pid 7916 != 7900, skipping destruction (fork without exec?)

**Comment by spooknik** [comment]
>  No such file or directory

It's either you don't have skse64_loader.exe installed or it's not installed correctly (not the same folder as SkyrimSE.exe.

Or you are currently not in the same folder that skse64_loader.exe is located in. 

You can try to run: 

`~/".steam/root/compatibilitytools.d/skyrim-proton/proton" run ~/".steam/steam/steamapps/common/Skyrim Special Edition/skse64_loader.exe"`

Let me know if that works.

**Comment by Xexman** [comment]
I do have skse in my steam common skyrim se folder. its in the same folder as skyrimSELauncher.exe.  still geting skipping destruction (fork without exec?)

**Comment by spooknik** [comment]
` skipping destruction (fork without exec?)`

Doesn't really mean anything either and can be ignored. As long as the game can run. 

Does it work if you run: `~/".steam/root/compatibilitytools.d/skyrim-proton/proton" run ~/".steam/steam/steamapps/common/Skyrim Special Edition/skse64_loader.exe"` ?

**Comment by Xexman** [comment]
[manjaro3@manjaro-pc Skyrim Special Edition]$ ~/".steam/root/compatibilitytools.d/skyrim-proton/proton" run 
bash: /home/manjaro3/.steam/root/compatibilitytools.d/skyrim-proton/proton: No such file or directory
pid 11042 != 11036, skipping destruction (fork without exec?)
[manjaro3@manjaro-pc Skyrim Special Edition]$ ~/".steam/steam/steamapps/common/Skyrim Special Edition/skse64_loader.exe
(it doesnts say or do anything at this point  just gives me  >[])

**Comment by spooknik** [comment]
> (it doesnts say or do anything at this point just gives me >[])

I think it's because you missed the quotation on the end. It should be:` ~/".steam/root/compatibilitytools.d/skyrim-proton/proton" run ~/".steam/steam/steamapps/common/Skyrim Special Edition/skse64_loader.exe"` 

not 

`~/".steam/root/compatibilitytools.d/skyrim-proton/proton" run ~/".steam/steam/steamapps/common/Skyrim Special Edition/skse64_loader.exe`

If it still doesn't work, try to confirm that this folder exists: ` /home/manjaro3/.steam/root/compatibilitytools.d/skyrim-proton/`and proton is there in this folder. Maybe something got screwed up with the install.

**Comment by Xexman** [comment]
[manjaro3@manjaro-pc Skyrim Special Edition]$ ~/".steam/steam/steamapps/common/Skyrim Special Edition/skse64_loader.exe"
bash: /home/manjaro3/.steam/steam/steamapps/common/Skyrim Special Edition/skse64_loader.exe: Permission denied
pid 11592 != 11587, skipping destruction (fork without exec?)
[manjaro3@manjaro-pc Skyrim Special Edition]$ /home/manjaro3/.steam/root/compatibilitytools.d/skyrim-proton/
bash: /home/manjaro3/.steam/root/compatibilitytools.d/skyrim-proton/: No such file or directory
pid 11595 != 11587, skipping destruction (fork without exec?)
[manjaro3@manjaro-pc Skyrim Special Edition]$

**Comment by spooknik** [comment]
> Permission denied

This isn't a good sign if you're getting permission denied from your own home folder. I have no idea what's going on. 

> No such file or directory

So yea, like I said in last message check that the file and folder you are actually trying to run exists. I'm sorry but if you can't troubleshoot simple errors like this, then this process is probably a bit too difficult for you. 

I'm going to make a video guide sometime this week, so maybe that can help troubleshoot some of the steps.

**Comment by Xexman** [comment]
Apparently /skyrim-proton/ doesn't exist  in /home/manjaro3/.steam/root/compatibilitytools.d/ (I had to go though manually to find that out). Honestly i dont know what to do at this point. SkyrimSE runs fine i just cant install anything that requires SKSE....

**Comment by RealGrep** [comment]
Yep, the script clearly didn't install it properly. Mine was all slapped directly into compatibilititools.d, not in a skyrim-proton directory. I created one, then copied it into the new directory.

I've utterly failed to get this work (and though I don't know Proton well, I know Wine and Linux extensively). I've gotten SKSE working on my own, previously, though for some reason it never saw the mods I copied from a Windows partition. I have noooo idea how to get Wrye Bash in there, either, with Proton. Instructions for that would be excellent.

Thanks for giving this a shot though! Seems to need a bit more work, but thanks, it's appreciated.

**Comment by Xexman** [comment]
I tryied reinstalling  skyrim-proton  and got this mv: cannot move '/tmp/skyrim-proton' to '/root/.steam/root/compatibilitytools.d/': No such file or directory

**Comment by RealGrep** [comment]
Xexman, looks like you're running it as root? That is a *terrible* idea.

**Comment by Xexman** [comment]
At this point am just trying anything i figure out to get it to work

**Comment by RealGrep** [comment]
Well, it explains getting "Permission denied" in your own home directory. There is no part of this particular procedure, or frankly, for any game at all, that should ever be done as root. Files created that way will only be accessible properly as root, not a normal user. Presumably, something was run as root, and it messed things up.
You will need to fix that as root. Not sure how the accounts and groups are set on your system, but you would need to run the 'sudo chmod' command to fix the bad permissions. Or zap it as root, and do it again without using root. Fixing that in text here with you would be quite difficult, so you'll need to learn how chmod works and find the files and directories that got messed up, I suspect.
There are very few times you should ever use the root account, or sudo (obviously upgrading your system, and things like that do need it). If a game required me to do so, I would literally not install that game, because there's no acceptable reason for it. Just wanted to make that clear so it prevents issues later. :)
EDIT: That said, the script did still put proton in the wrong place.

**Comment by Xexman** [comment]
I only tried reinstalling proton as root the first time i didnt.  so the Permission denied" in your own home directory was caused by something else

**Comment by Xexman** [comment]
I have all ready boot my system to a preveise state before the reinstall as well

**Comment by spooknik** [comment]
Script has been updated to install to the correct location. 

@Xexman I'm sorry but I can't help you figure out which files and folders in your home directory have been affected by running things as sudo. 

@RealGrep You can't just dump mods into Skyrim's data folder, something needs to activate them in the load order. Either Wrye Bash, Mod organizer, or even Skyrim itself via the 'Mods' menu in game. You can run other exe's from xterm the same way you would run skse64_loader.exe. Just replace the exe you want to run. Wrye Bash has a standalone .exe just put that somewhere in your home folder and you can point proton to running that exe.` ~/".steam/root/compatibilitytools.d/skyrim-proton/proton" run ~/"Documents/Mopy/Wrye Bash.exe"`

In summary, I thought this wasn't so complicated and I think I need to explain things in a better way. I'll make a video and hopefully that can help :+1: 

I'm closing this issue now. Please if anyone has further problems just open a new issue.
